spec/factories.rb

```diff
@@ -1214,4 +1224,25 @@
     feedback { Faker::Lorem.paragraph(sentence_count: 3) }
     consent_to_be_contacted { true }
   end
+
+  factory :interview_audit, class: 'Audited::Audit' do
+    action { 'create' }
+    user { create(:provider_user) }
+    version { 1 }
+    request_uuid { SecureRandom.uuid }
+    created_at { Time.zone.now }
+
+    transient do
+      application_choice { create(:application_choice, :awaiting_provider_decision) }
```

Comment:
Can we change this to `build` instead of `create`. I think we should in general avoid creating object in the factories, but building them should enable creation anyway if `create` is called.

Reply:
@despo https://github.com/DFE-Digital/apply-for-teacher-training/pull/3849/commits/ef88aab93675aa9dfe0557eea80c7894d98c383b was simple enough to stub everything